src/pages/Signup.js

```diff
@@ -1,6 +1,107 @@
+    const confirmPasswordInput = useRef(null)
+    const firstNameInput = useRef(null)
+    const lastNameInput = useRef(null)
+    const navigate = useNavigate()
+
+
+    async function CreateAccountRequest(accountPayload){
+
+        let response = {}
+
+        response = await apiUtil('post', '/users', {}, accountPayload)
+        console.log(" ==response: ", response)
+        console.log(" ==response: ", response.message)
+
+        setError(response.error)
+        setMessage(response.message)
+        setEmail("")
+        setPassword("")
+        setConfirmPassword("")
+        setFirstName("")
+        setLastName("")
+
+        if(response.status == 201){
+            navigate("/login")
+        }
+
+    }
+
+    function createAccountStaging(){
+        event.preventDefault()
+
+        const accountInformation = {
+            email: email,
+            rawPassword: password,
+            confirmedPassword: confirmPassword,
+            firstName: firstName,
+            lastName: lastName
+        }
+
+        CreateAccountRequest(accountInformation)
+        emailInput.current.value = ''
+        passwordInput.current.value = ''
+        confirmPasswordInput.current.value = ''
+        firstNameInput.current.value = ''
+        lastNameInput.current.value = ''
+    }
+
+    return (
+        <div>
+            <h1>Sign up to use MyClassroom</h1>
+            <form>
+                <h2>Sign up with your email address</h2>
+                <div>
+                    <label>
+                        <span>What's your email?</span>
+                    </label>
+                    <input ref={emailInput} onChange={ (event) => setEmail(event.target.value) } placeholder="Enter your email." />
+                </div>
+
+                <div>
+                    <label>
+                        <span>Create a password</span>
+                    </label>
+                    <input ref={passwordInput} onChange={ (event) => setPassword(event.target.value) } placeholder="Create your password."/>
+                    <input ref={confirmPasswordInput} onChange={ (event) => setConfirmPassword(event.target.value) } placeholder="Confirm your password."/>
+                </div>
+
+                <div>
+                    <label>
+                        <span>What should we call you?</span>
+                    </label>
+                    <input ref={firstNameInput} onChange={ (event) => setFirstName(event.target.value) } placeholder="First name"/>
+                    <input ref={lastNameInput} onChange={ (event) => setLastName(event.target.value) } placeholder="Last name"/>
+                </div>
+
+                <div>
+                    <button onClick={createAccountStaging} type="submit">
+                        <span>Sign Up</span>
+                    </button>
+                </div>
+            </form>
+
+            <div>
+                { message !== "" && <Notice message={message} error={error}/> }
```

Comment:
The notice object changed to expect a status prop instead of a boolean error prop

Reply:
Here is the error message im getting, and here is the code.
![image](https://github.com/CS461PollingApplication/my-classroom-fe/assets/94574064/fa379dc7-8a1a-43d9-9863-7dbf7065dfc5)

![image](https://github.com/CS461PollingApplication/my-classroom-fe/assets/94574064/b2c58bd6-f756-483a-93cd-47216adf7e7b)
Here is the code to show.

Should be good!

Reply:
Oh sorry, I should have communicated better about what I meant. There's a different display color and symbol for errors, that way it's red instead of blue and a little more intuitive that it's a user error

Reply:
Ok I just copied what was done on the other files. Hopefully this is it. Apologies in advance if this is still wrong...